lgc/builder/InOutBuilder.cpp

```diff
@@ -1580,6 +1644,40 @@ void InOutBuilder::markBuiltInOutputUsage(BuiltInKind builtIn, unsigned arraySiz
     break;
   }
 
+  case ShaderStageMesh: {
+    switch (builtIn) {
+    case BuiltInPointSize:
+      usage.mesh.pointSize = true;
+      break;
+    case BuiltInPosition:
+      usage.mesh.position = true;
+      break;
+    case BuiltInClipDistance:
+      usage.mesh.clipDistance = std::max(usage.mesh.clipDistance, arraySize);
+      break;
+    case BuiltInCullDistance:
+      usage.mesh.cullDistance = std::max(usage.mesh.cullDistance, arraySize);
+      break;
+    case BuiltInPrimitiveId:
+      usage.mesh.primitiveId = true;
+      break;
+    case BuiltInViewportIndex:
+      usage.mesh.viewportIndex = true;
+      break;
+    case BuiltInLayer:
+      usage.mesh.layer = true;
+      break;
+    case BuiltInCullPrimitive:
+      usage.mesh.cullPrimitive = true;
+      break;
+    case BuiltInPrimitiveShadingRate:
+      usage.mesh.primitiveShadingRate = true;
+      break;
+    default:
+      break;
+    }
+  }
```

Comment:
@amdrexu You are missing a "break;" here. I just got a warning building it with clang:

```
/Users/tpr/amd/git/sw/drivers/xgl/icd/api/compiler/lgc/builder/InOutBuilder.cpp:1732:3: warning: unannotated fall-through between switch labels [-Wimplicit-fallthrough]
  case ShaderStageFragment: {
  ^
```

Reply:
Thank you. I did make a mistake here. Will correct it.

Reply:
Luckily, Sebastian's change fixes this for me: https://github.com/GPUOpen-Drivers/llpc/pull/1987. Just need fetch the latest codes.